default.nix

```diff
@@ -43,8 +43,17 @@ in
 { ... }@args:
 # We will instantiate the default-nix template with the
 # nix/pkgs.nix file...
-commonLib.nix-tools.default-nix ./nix/pkgs.nix args
+let
+  d = commonLib.nix-tools.default-nix ./nix/pkgs.nix args;
+in
 # ... and add additional non-haskell packages we want to build on CI:
-// {
-
-}
+  d // {
+    shell = d.nix-tools._raw.shellFor {
```

Comment:
You can drop the `_raw`: https://github.com/input-output-hk/iohk-nix/blob/master/nix-tools-default.nix#L33

Reply:
Oh good, thanks.